xml/xml.go

```diff
@@ -0,0 +1,39 @@
+package xml
```

Comment:
Keeping this package name means that we need to alias when also importing `encoding/xml`.